scripts/process.py

```diff
@@ -1,27 +1,47 @@
-import urllib
-import zipfile
+import io
+import os
 import csv
+import zipfile
+import requests
 
-version = '19'
-source = "http://www.eea.europa.eu/data-and-maps/data/\
-european-union-emissions-trading-scheme-eu-ets-data-from-citl-7/\
-eu-ets-data-download-latest-version/citl_v"+version+".zip/at_download/file"
-zip = 'tmp/citl.zip'
-csvfile = 'tmp/CITL_v'+version+'.csv'
-out_path = 'data/eu-ets.csv'
+source = "https://sdi.eea.europa.eu/datashare/s/SxpWTfMxkZSrpBo/download"
```

Comment:
@gradedSystem let's keep an eye on this uri just to ensure it doesn't change month to month. My concern is that it has some hash in the uri.